_configs/config.json

```diff
@@ -11,14 +11,14 @@
     "browser/extensions/formautofill/locales",
     "browser/extensions/report-site-issue/locales",
     "browser/locales",
+    "browser/pdfviewer/locales",
```

Comment:
This one is "interesting". From [toolkit](https://searchfox.org/mozilla-central/source/toolkit/locales/l10n.toml):

```
[[paths]]
    reference = "browser/locales/en-US/pdfviewer/**"
    l10n = "{l}browser/pdfviewer/**"
```

In the local config (which I use in this script)

```
{ reference = "browser/pdfviewer/**", l10n = "{l}browser/pdfviewer/**" },
```

Should this be treated as a bug in `update.py`?

Note that the folder in question is only [available in esr115](https://searchfox.org/mozilla-esr115/source/browser/locales/en-US/pdfviewer).

Reply:
Actually, `update.py` is fine, since `browser/pdfviewer` since that path exists.

Should I create a special case for this path to ignore it, or should we just live with this non-existing path?